swan-lake/learn/using-the-cli-tools.md

```diff
@@ -14,25 +14,26 @@ redirect_from:
 
 ## Using the Ballerina Tool
 
-The Ballerina Tool is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina projects and modules, test, build, and run programs, etc.
+The Ballerina Tool is a command-line tool for managing Ballerina source code. It helps you to manage Ballerina packages and modules, test, build, and run programs, etc.
 
 It also enables you to easily install, update, and switch among Ballerina distributions. 
 
-In the CLI, execute the `ballerina help` command to view all the actions you can perform with the Ballerina Tool as shown below:
+In the CLI, execute the `bal help` command to view all the actions you can perform with the Ballerina Tool as shown below:
 
 ```sh
-→ ballerina help 
+→ bal help
 NAME
-       The Ballerina build tool
+       The bal build tool
 
 SYNOPSIS
-       ballerina <-v | --version>
-       ballerina [command] <-h | --help>
-       ballerina <command> [<args>]
+       bal <-v | --version>
+       bal [command] <-h | --help>
+       bal <command> [<args>]
 
 
 DESCRIPTION
-       Ballerina is a statically typed, concurrent programming language, focusing on network interaction and structured data. It is intended to be the core of a language-centric middleware platform. It has all the general-purpose
+       Ballerina is a statically typed, concurrent programming language, focusing on network interaction and structured
```

Comment:
```suggestion
       Ballerina is a statically-typed, concurrent programming language focusing on network interaction and structured
```

Reply:
@praneesha this as well is a CLI output generated by the `bal help` command.
Need to specifically address these suggested CLI output text issues in the [ballerina-lang repo](https://github.com/ballerina-platform/ballerina-lang).

Reply:
Will be address at https://github.com/ballerina-platform/ballerina-lang/issues/28138